api/src/mappers.js

```diff
@@ -10,18 +10,16 @@ const PARAMETERS = {
  * @return {Object}                  Object containing valid and invalid parameters.
  */
 export const parseRequest = (request) => {
-  const { url } = request;
-  const whitelist = PARAMETERS[url];
-  const { params } = request;
-  return Object.keys(params).reduce(
-    ({ parameters }, param) => {
-      const { valid: validParameters, invalid: invalidParameters } = parameters;
-      if (whitelist.includes(param))
-        return { parameters: { valid: { ...validParameters, param }, invalid: invalidParameters } };
-      return { parameters: { valid: validParameters, invalid: { ...invalidParameters, param } } };
+  const { url } = request.raw;
+  const whitelist = PARAMETERS[url.split('?')[0]];
```

Comment:
Nit: is there a way to get the base url (i.e. without splitting on the query param)?

Reply:
That's a good question! I had a look and both the Fastify request object and the raw request object don't seem to have it conveniently available without the query param. The hostname portion is, but that's not so useful for us in this instance 😿